HeadersImports.md

```diff
@@ -34,6 +34,10 @@ In Swift Package Manager, it's a library target.
   should be located among the source files. [Xcode](https://stackoverflow.com/a/8016333) refers to
   these as "Project Headers".
 
+* *Library C++ Internal Headers* - In CocoaPods, C++ internal headers should not be included
```

Comment:
While this specifies the consequence (that C++ headers should be included), it doesn't give a rationale for why that's the case. The issue is that C++ does not assume a global header map, so if filenames are qualified at all, it's generally by directory, not a filename prefix like in Objective-C. Is it worth adding something here to make this clear?

Reply:
Done